core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -281,16 +281,8 @@ object Boot extends IOApp with LazyLogging {
       val requesterPaysRole = gcsConfig.getString("requesterPaysRole")
       val projectTemplate = ProjectTemplate(projectOwners, projectEditors)
 
-      val notificationPubSubDAO = new org.broadinstitute.dsde.workbench.google.HttpGooglePubSubDAO(
-        clientEmail,
-        pathToPem,
-        appName,
-        serviceProject,
-        workbenchMetricBaseName = metricsPrefix
-      )
-
       val notificationDAO = new PubSubNotificationDAO(
-        notificationPubSubDAO,
+        pubSubDAO,
```

Comment:
with the library change in this PR, could you also update the constructors for `pubSubDAO` and `importServicePubSubDAO` at line 187 to use the non-deprecated signature?

Reply:
Done in 6ab019f10